contrib/zep/src/window.cpp

```diff
@@ -164,10 +162,19 @@ void ZepWindow::UpdateAirline()
 
         auto cursor = BufferToDisplay();
         m_airline.leftBoxes.push_back(AirBox{ m_pBuffer->GetDisplayName(), FilterActiveColor(m_pBuffer->GetTheme().GetColor(ThemeColor::AirlineBackground)) });
-        m_airline.leftBoxes.push_back(AirBox{ std::to_string(cursor.x) + ":" + std::to_string(cursor.y), m_pBuffer->GetTheme().GetColor(ThemeColor::TabActive) });
+
+        if (m_displayMode == DisplayMode::Vim && m_pBuffer->GetMode()->GetCursorType() != CursorType::None)
+        {
+            m_airline.leftBoxes.push_back(AirBox{ std::to_string(cursor.x) + ":" + std::to_string(cursor.y), m_pBuffer->GetTheme().GetColor(ThemeColor::TabActive) });
+        }
+        else
+        {
+            m_airline.rightBoxes.push_back(
+                AirBox{ fmt::format("{}:{}", cursor.x + 1, cursor.y + 1), m_pBuffer->GetTheme().GetColor(ThemeColor::TabActive) });
+        }
 
 #ifdef _DEBUG
-        m_airline.leftBoxes.push_back(AirBox{ "(" + std::to_string(GetEditor().GetDisplay().GetPixelScale().x) + "," + std::to_string(GetEditor().GetDisplay().GetPixelScale().y) + ")", m_pBuffer->GetTheme().GetColor(ThemeColor::Error) });
+        //m_airline.leftBoxes.push_back(AirBox{ "(" + std::to_string(GetEditor().GetDisplay().GetPixelScale().x) + "," + std::to_string(GetEditor().GetDisplay().GetPixelScale().y) + ")", m_pBuffer->GetTheme().GetColor(ThemeColor::Error) });
```

Comment:
should this remain in for debug builds? it looks useful

Reply:
We don't change pixel scale, so it is always 1.0